Project.toml

```diff
@@ -8,6 +8,7 @@ Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 MLJModelInterface = "e80e1ace-859a-464e-9ed9-23947d8ae3ea"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+UnsafeArrays = "c4a57d5a-5b31-53a6-b365-19f8c011fbd6"
```

Comment:
`UnsafeArrays` version should be added to `compat`, otherwise you wouldn't be able to register new version.